crypto/include/internal/bn_int.h

```diff
@@ -60,4 +60,12 @@ void bn_set_static_words(BIGNUM *a, BN_ULONG *words, int size);
  */
 int bn_set_words(BIGNUM *a, BN_ULONG *words, int num_words);
 
+/*
+ * BN_mod_mul_montgomery without bn_correct_top.
```

Comment:
I would suggest a comment here like:

Note that some BIGNUM functions assume bn_correct_top was run. For now, the outputs of these functions may not be returned out of public APIs and may only be passed into other functions known to support non-minimal BIGNUMs. Once https://github.com/openssl/openssl/issues/6640 is completed, this restriction may be lifted.